src/_layouts/rss.liquid

```diff
@@ -0,0 +1,42 @@
+---
+layout: null
+spec: http://cyber.law.harvard.edu/rss/rss.html
```

Comment:
No - it's more of a comment so that people updating the feed can easily refer back to the RSS spec.